locale/telemosaic.de.tr

```diff
@@ -0,0 +1,45 @@
+# textdomain: telemosaic
+##[ beacon.lua ]##
+Protected Telemosaic Beacon=Geschützte Telemosaik-Leuchtfeuer
+Telemosaic Beacon=Telemosaik-Leuchtfeuer
+Telemosaic Beacon Active (you hacker you!)=Telemosaik-Leuchtfeuer aktiv (du Hacker du!)
+Protected Telemosaic Beacon Active (you hacker you!)=Geschützte Telemosaik-Leuchtfeuer aktiv (du Hacker du!)
+Telemosaic Beacon Error (you hacker you!)=Telemosaik-Leuchtfeuer fehler (du Hacker du!)
+Protected Telemosaic Beacon Error (you hacker you!)=Geschützte Telemosaik-Leuchtfeuer fehler (du Hacker du!)
```

Comment:
`Fehler` would be a noun, while I intended an adjective. Is it not possible to correctly use an adjective here? That's how I interpret the English, though it's not entirely grammatical English. To be nitpicky, it would be "Protected Telemosaic in error state (you hacker, you!)"

Reply:
yes `Fehler` is a noun but the adjective would be `fehlerhaft`. `fehler` is not a valid word